### DISCUSSION TITLE
can ripgrep report the pattern that matches when given multiple patterns?

### DISCUSSION THREAD

**Discussion by ccaglak** [BurntSushi/ripgrep#2931] [Q&A] [answered]
Hi All, i am trying to figure out if i could have a output in this format

```
 "rg",
      "--multiline",
      "-e",
      "abc",
      "-e",
      "def",

```
identify which pattern is used 
line:column:type:match
 9:34:abc:abc("")

thanks in advance

**Comment by BurntSushi** [comment, marked as answer]
No, ripgrep does not support reporting the pattern that matches.

**Comment by ccaglak** [comment]
thank you, cool project
